src/test/kotlin/com/zepben/evolve/services/network/tracing/phases/PhaseStepTest.kt

```diff
@@ -17,14 +17,14 @@ import org.junit.jupiter.api.extension.RegisterExtension
 import org.mockito.kotlin.mock
 import com.zepben.evolve.cim.iec61970.base.wires.SinglePhaseKind as SPK
 
-class PhaseStepTest {
+internal class PhaseStepTest {
 
     @JvmField
     @RegisterExtension
-    var systemOutRule: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+    var systemOut: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
```

Comment:
todo: should have been renamed `systemErr`

Reply:
Renamed as such.